packages/component-library/draft/Kaizen/SplitButton/styles.scss

```diff
@@ -65,15 +67,58 @@ $button-width: 124px;
   // Disabled
   &:disabled,
   &.disabled {
-    background: $kz-color-white;
-    border-color: $kz-color-wisteria-200;
-    color: $kz-color-wisteria-300;
+    background: $disabled-background;
+    border-color: $disabled-border-color;
+    color: $disabled-color;
   }
 }
 
-.button {
+.root {
+  @include ca-type-block(inline-block);
+}
+
+.buttonReset {
+  appearance: none;
+  display: inline;
+  background: transparent;
+  color: inherit;
+  font: inherit;
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+  border: 0;
+}
+
+.buttonDefault {
+  @include button-mixing($background: $kz-color-white,
+    $border-color: $kz-color-wisteria-200,
+    $color: $kz-color-wisteria-800,
+    $hover-background: $kz-color-stone,
+    $hover-border-color: $kz-color-wisteria-400,
+    $active-background: $kz-color-ash,
+    $active-border-color: $kz-color-wisteria-500,
+    $disabled-background: $kz-color-white,
+    $disabled-border-color: $kz-color-wisteria-200,
+    $disabled-color: $kz-color-wisteria-300);
+}
+
+.buttonPrimary {
+  // equal to add-tint($ca-palette-seedling, 10%). Temporary until we upgrade to Zen (Wisteria)
```

Comment:
added this comment that is the same that Michael added in [this](https://github.com/cultureamp/kaizen-design-system/commit/1bc36311c7ba674f8525fbe7b15e2c705caeca8b) commit  in [this](https://github.com/cultureamp/kaizen-design-system/commit/1bc36311c7ba674f8525fbe7b15e2c705caeca8b#diff-5e77baee7dee0b91c4ef99b35bbc011b) file